index.html

```diff
@@ -307,5 +307,31 @@ <h3 class="skill-title">Skills</h3>
         </li>
       </ul>
     </section>
+
+    <section class="form-section">
+      <div class="form-section-wrapper">
+        <p class="form-description">
+          I'm always interested<br>in hearing about new<br> projects, so if you'd<br>like to chat please<br> get in touch.
+        </p>
+      </div>
+      <form class="contact-form" action="https://formspree.io/f/mknayjll" method="POST">
+        <input type="text" name="username" maxlength="30" placeholder="Full Name" required><br><br>
+        <input type="email" name="email" placeholder="Email address" required><br><br>
+        <textarea maxlength="500" placeholder="Write me something..." name="message-box"  required></textarea><br><br>
+        <input type="submit" value="Get in touch">
+      </form>
+    </section>
+
+    <section class="footer-section">
+    <div class="form-line"></div>
+      <div class="footer-section-wrapper">
+          <img
+            class="footer-link-img"
+            src="images/Social Media.png"
+            alt="Social Media"/>
+          <div class="form-base-line"></div>
+      </div>
```

Comment:
- [ ] However much your design matches the Figma template design, you did not use proper semantics to layout the social icons of the footer. You should make sure that every social icon is attached to your personalized link. This is to ensure that the recruiters other portifolio visitors can easily contact you. **Hint:** You may need to use `ul` and `li`.